_tools/lint_plugin.ts

```diff
@@ -59,6 +59,10 @@ export default {
     // https://docs.deno.com/runtime/contributing/style_guide/#do-not-depend-on-external-code.
     "no-external-code": {
       create(context) {
+        if (context.filename.includes("_tools")) {
+          // Tools are allowed to use external code
+          return {};
+        }
```

Comment:
I used ignore lint directives instead of plugin logic so that this plugin can be used for other codebases, as per the original intention of #655. Either way, we can address this if that ends up happening.

Reply:
Isn't [this rule](https://docs.deno.com/runtime/contributing/style_guide/#std) specific to std? I guess this rule only makes sense as private rule here